Servers/TTYServer/main.cpp

```diff
@@ -0,0 +1,22 @@
+#include <AK/Assertions.h>
+#include <LibCore/CFile.h>
```

Comment:
This doesn't get used, so we don't need to link with `-lcore` either :)

Reply:
Guess who copy/pasted this from SystemServer? :D